watertap/unit_models/electrodialysis_0D.py

```diff
@@ -2294,6 +2348,47 @@ def calculate_scaling_factors(self):
                 )
             iscale.set_scaling_factor(self.pressure_drop_total, sf)
 
+        if hasattr(self, "current_dens_lim_ioa"):
+            if iscale.get_scaling_factor(self.current_dens_lim_ioa) is None:
+                if (
+                    self.config.limiting_current_density_method
+                    == LimitingCurrentDensityMethod.InitialValue
+                ):
+                    sf = self.config.limiting_current_density_data**-1
+                    iscale.set_scaling_factor(self.current_dens_lim_ioa, sf)
+                elif (
+                    self.config.limiting_current_density_method
+                    == LimitingCurrentDensityMethod.Empirical
+                ):
+                    sf = 25**-1 * sum(
+                        iscale.get_scaling_factor(
+                            self.diluate.properties_in[0].conc_mol_phase_comp["Liq", j]
+                        )
+                        ** 2
+                        for j in self.cation_set
+                    ) ** (0.5 * 0.5)
+                    iscale.set_scaling_factor(self.current_dens_lim_ioa, sf)
+                elif (
+                    self.config.limiting_current_density_method
+                    == LimitingCurrentDensityMethod.Theoretical
+                ):
+                    sf = (
+                        iscale.get_scaling_factor(self.N_Sh)
+                        * iscale.get_scaling_factor(self.diffus_mass)
+                        * iscale.get_scaling_factor(self.hydraulic_diameter) ** -1
+                        * 96485**-1
```

Comment:
Just Constants.faraday_constant?

Reply:
No difference and save the computational effort to strip the unit.